website/docs/add-secure-apps/providers/ssf/index.md

```diff
@@ -36,12 +45,19 @@ Note that authentik doesn't specify which events to subscribe to; instead the ap
 
 ### authentik (the transmitter)
 
-To configure authentik as a shared signals transmitter, the authentik Admin [creates a new provider](./create-ssf-provider), selecting the type "SSF", to serve as the backchannelprovider for the application.
+To configure authentik as a shared signals transmitter, navigate to your authentik instance's Admin interface. [Create a new provider](./create-ssf-provider), selecting the type "SSF", to serve as the backchannel provider for the application.
 
-When creating the SSF provider you will need to select a signing key. This is the key that the Security Event Tokens (SET) is signed with.
+When creating the SSF provider, you will need to select a key to sign Security Event Tokens (<abbr>SET</abbr>). This key is used to sign the SETs that are sent to the application. The application will use this key to verify the SETs.
 
-Optionally, you can specify a event retention time period: this value determines how long events are stored for. If an event could not be sent correctly, and retries occur, the event's expiration is also increased by this duration.
+You may also specify a event retention time period. If an event could not be sent correctly, and retries occur, the event's expiration is also increased by this duration.
+
+:::info{title="Events and SETs"}
```

Comment:
I'm glad to know about the ability to give a specific title to an info block, but I don't want to use one here. Especially not this title, becaue it makes it sound like the info block is only about SSF Ents and SETs, when in reality it is pointing out that ther are other types of events, unrelated to these SSF events.

Reply:
Also we both missed "an" versus "a" before "event". :-)